packages/synapse-react-client/src/components/GenericCard/PortalDOI/PortalDOIUtils.ts

```diff
@@ -33,12 +33,10 @@ export function useShowDoiCardLabel(opts: {
 export function getCandidateDoiId(args: {
   /** Configuration to display a DOI, as well as the ability to create one for users with such permission */
   portalDoiConfiguration?: TableToGenericCardMapping['portalDoiConfiguration']
-  /** Mapping of column name to row data index */
-  schema: Record<string, number>
-  /** The row data */
-  data: string[]
+  /** A record that represents a table row or DOI lookup, mapping from column name to value */
+  data: Record<string, string | null>
 }): string | undefined {
```

Comment:
In the 'tombstone' case, we don't actually have a row of data. We have search params, which is the direct mapping of column name to cell value.

So I changed this API to accommodate this case. In the existing place where we are using row data to generate this ID, we will transform the data to a record.